### PR TITLE
LPS-99441 | LPS-103592 Regen

### PR DESCRIPTION
**Pull request by gabrielpramos**

### CODE REVIEW

**Comment by gabrielpramos** [comment]
ci:test:sf

**Comment by gabrielpramos** [comment]
ci:test:relevant

**Comment by albinojunior** [comment]
Just started reviewing :)

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/performance-by-assignee-page/PerformanceByAssigneePageTable.es.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+					<ClayTable.Cell headingCell style={{width: '20%'}}>
+						<ListHeadItem
+							name="taskCount"
+							title={Liferay.Language.get('completed-tasks')}
+						/>
+					</ClayTable.Cell>
+
+					<ClayTable.Cell
+						headingCell
+						headingTitle
```
Comment (albinojunior) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/performance-by-assignee-page/PerformanceByAssigneePageTable.es.js]: @gabrielpramos remove unnecessary property "headingCell" and resend.

**Pull request closed without merging**